### PR TITLE
#13967: Support for clip_bw

### LINKED ISSUE #13967: Support for clip_bw
With respect to comment [here](https://github.com/tenstorrent/tt-metal/pull/13850#pullrequestreview-2376894774), clip_bw support is required

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
#13967 
Support for clip_bw

### CODE REVIEW

File: tests/ttnn/unit_tests/operations/eltwise/backward/test_backward_clip.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        (0.0, 1.0),
+    ],
+)
+def test_unary_bw_clip(input_shapes, min, max, device):
+    in_data, input_tensor = data_gen_with_range(input_shapes, -100, -1, device, True)
+    grad_data, grad_tensor = data_gen_with_range(input_shapes, -10, -1, device, True)
+    if min is None and max is None:
+        with pytest.raises(RuntimeError, match="Only one of 'min' or 'max' can be None. Please provide one value"):
+            ttnn.clip_bw(grad_tensor, input_tensor, min, max)
+        assert True
```
Comment (Aswinmcw) [inline tests/ttnn/unit_tests/operations/eltwise/backward/test_backward_clip.py:41]: skip instead of error?

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by umadevimcw — APPROVED** [review]

**Comment by umadevimcw** [comment]
Can we write sweep test as well?

**Review by eyonland — APPROVED** [review]

File: tests/ttnn/unit_tests/operations/eltwise/backward/test_backward_clip.py (hunk 1 above)
Comment (TT-BrianLiu) [reply tests/ttnn/unit_tests/operations/eltwise/backward/test_backward_clip.py:41]: This is a test that is testing the expected failure of the op? Maybe mark with xfail.

**Review by TT-BrianLiu — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11595975874 - passed

File: tests/ttnn/unit_tests/operations/eltwise/backward/test_backward_clip.py (hunk 1 above)
Comment (VirdhatchaniKN) [reply tests/ttnn/unit_tests/operations/eltwise/backward/test_backward_clip.py]: Ive addressed this comment.

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11640066114 - passed

**Pull request merged**